### PR TITLE
Enable scl globally

### PR DESCRIPTION
**Pull request by jhadvig**
@soltysh PTAL

### CODE REVIEW

**Comment by soltysh** [comment]
You should also remove `0.10/contrib/lib/nodejs_context` file.

**Comment by soltysh** [comment]
Other than that LGTM.

File: 0.10/.sti/bin/assemble [hunk 1]
```diff
@@ -1,11 +1,13 @@
-#!/bin/bash -e
+#!/bin/bash
 
 # Prevent running assemble in builders different than official STI image.
 # The official nodejs:0.10-onbuild already run npm install and use different
 # application folder.
 [ -d "/usr/src/app" ] && exit 0
 
-source "../lib/nodejs_context"
+source scl_source enable nodejs010
```
Comment (mfojtik) [inline 0.10/.sti/bin/assemble]: why you need to do this when you enable it in bashrc?

File: 0.10/.sti/bin/run [hunk 2]
```diff
@@ -1,4 +1,8 @@
-#!/bin/bash -e
+#!/bin/bash
+
+source scl_source enable nodejs010
```
Comment (mfojtik) [inline 0.10/.sti/bin/run]: same here :)

File: 0.10/.sti/bin/assemble (hunk 1 above)
Comment (jhadvig) [inline 0.10/.sti/bin/assemble]: bashrc is not loaded in the STI scripts, only when enter the container

**Comment by jhadvig** [comment]
@soltysh updated :) thanks for noticing, guess I overlooked it due to the hunger :hamburger:

File: 0.10/.sti/bin/assemble (hunk 1 above)
Comment (mfojtik) [inline 0.10/.sti/bin/assemble]: source it then

Comment (soltysh) [inline 0.10/.sti/bin/assemble]: Not true, I'm not sourcing it anywhere in [sti-python](https://github.com/openshift/sti-python), `.bashrc` is loaded upon all `bash` invocations and that's what STI is doing.

**Comment by soltysh** [comment]
LGTM.

**Comment by mfojtik** [comment]
@soltysh I suggested to source it to avoid having scl commands and logic all over the place.... we should enable it in one place and then just source it (bashrc).

**Comment by mfojtik** [comment]
@soltysh @jhadvig my point is to allow disabling scl easy (if you want to). in that case I have to comment it just in one place and not in all scripts.

**Comment by soltysh** [comment]
I misread that, but yeah that'd be good. You're welcome to create follow up
PRs for all the images :D

**Pull request merged**